backend/app/helpers/backend/forms_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Backend
+  module FormsHelper
+    def is_cover_photo?(media_content, photo)
+      media_content.try(:main_photo_id) == photo.try(:photo_id) if media_content.try(:main_photo_id).present?
```

Comment:
suggest replacing with:

`media_content.main_photo_id.present? && photo.photo_id.present? && media_content.main_photo_id == photo.photo_id`

No need to use try if you are passing an object and checking an attribute of that object, I think.